_posts/2021-12-14-announce-github-naming-change.md

```diff
@@ -0,0 +1,18 @@
+---
+layout: post
+title: "Announcing Github renaming change"
+date:   2021-12-14 00:00:00 -0600
+synopsis: "Github is changing master to main, and we are, too."
+author_info:
+  name:
+    first: "Lorina"
+    last: "Poland"
+  picture: "assets/images/stargate-profile.png"
+---
+
+In October 2020 , all new source code repositories created on GitHub will be named 'main' instead of `master` as part of the company's effort to remove unnecessary references to slavery and replace them with more inclusive terms. This "fix forward first" approach is the first step in making systemic changes to the naming conventions. 
```

Comment:
extra space before the comma after 2020

Reply:
should the `main` and `master` use backticks or single quotes consistently with each other?

Reply:
Done, and done.